debian/bionic/foreman/rules

```diff
@@ -5,6 +5,8 @@
 # DEB_VERBOSE_ALL=1
 # DH_VERBOSE=1
 
+DEB_BUILD_OPTIONS=noautodbgsym
```

Comment:
while this doesn't speed up the build, the dbgsym package is not generated and thus doesn't take up space on the mirror